Source/TrackableEntities.Client/EntityBase.cs

```diff
@@ -23,11 +23,24 @@ public abstract partial class EntityBase : INotifyPropertyChanged, ITrackable, I
             (Expression<Func<TResult>> property)
         {
             string propertyName = ((MemberExpression)property.Body).Member.Name;
+            NotifyPropertyChanged(propertyName);
```

Comment:
Why would you want to do that for?
I think it's best if we have a virtual `NotifyPropertyChanged` method that all calls tunnel thru it (or we add an additional empty virtual method `OnNotifyPropertyChanged` that's called just after the notification happens.
It's best to give more flexibility and option to intercept the property notification for subclasses hence making it `protected`, so we're not tied to subscribing to event and caring about memory leaks and unsubscribing.